contracts/SuperpowerNFTBase.sol

```diff
@@ -196,7 +198,12 @@ abstract contract SuperpowerNFTBase is
     emit GameSet(game_);
   }
 
-  // locks
+  // ILockable
+  //
+  // When a contract is locked, only the locker is approved
+  // Then advantage of locking an NFT instead of staking is that
+  // The owner keeps the ownership of it and can use that, for example,
```

Comment:
Minor typo:
// Then advantage of locking an NFT instead of staking is that
// The owner keeps the ownership of it and can use that, for example,
should be
// The advantage of locking an NFT instead of staking is that
// The owner keeps its ownership and can use it, for example,

Reply:
fixed